Source/Core/DolphinQt2/Config/GeckoCodeWidget.cpp

```diff
@@ -220,10 +223,20 @@ void GeckoCodeWidget::SaveCodes()
   game_ini_local.Save(File::GetUserPath(D_GAMESETTINGS_IDX) + m_game_id + ".ini");
 }
 
+bool helpOrderCodes(Gecko::GeckoCode code1, Gecko::GeckoCode code2) { // to sort the codes alphabetically
```

Comment:
This function isn't necessary and a lambda function should be passed to std::sort. i.e.

```cpp
std::sort(m_gecko_codes.begin(), m_gecko_codes.end(),
          [](const auto& lhs, const auto& rhs) { return lhs.name < rhs.name; });
```